backend/src/routes/api/images/imageUtils.ts

```diff
@@ -91,6 +90,18 @@ const getImageStreams = async (
   return await requestPromise;
 };
 
+const isBYONImage = (imageStream: ImageStream) =>
+  imageStream.metadata.labels?.['app.kubernetes.io/created-by'] === 'byon';
+
+const getBYONImageErrorMessage = (imageStream: ImageStream) => {
+  // there will be always only 1 tag in the spec for BYON images
+  // status tags could be more than one
+  const activeTag = imageStream.status?.tags?.find(
+    (statusTag) => statusTag.tag === imageStream.spec.tags?.[0].name,
+  );
+  return activeTag?.items === null ? activeTag.conditions?.[0].message : undefined;
```

Comment:
Just out of curiosity, cause i couldn't validate it checking the spec. Why do you need items to be null to get the error message? Is it possible to have an issue with the status filled with items? Why not try to access `activeTag.conditions?.[0].message` and fallback to undefined if you could not get it?

Reply:
hmm, good catch. I will change it. I cannot validate the status spec too.